en/operations/configuration-server.html

```diff
@@ -204,24 +203,26 @@ <h2 id="start-sequence">Start sequence</h2>
 
 
 
-<h2 id="scaling-up">Scaling Up</h2>
+<h2 id="scaling-up">Scaling up</h2>
 <p>
-  Add config server nodes for increased fault tolerance. Procedure:
+  Do this by adding nodes one by one. Add a config server node for increased fault tolerance or when replacing a node. Procedure:
 </p>
 <ol>
   <li>
-    Install <em>vespa</em> on the new config server nodes
+    Install <em>vespa</em> on new config server node.
   </li><li>
-    Add config server hosts to <em>VESPA_CONFIGSERVERS</em> on <em>all</em> the nodes
+    Add config server node to <em>VESPA_CONFIGSERVERS</em> on all nodes (both config server nodes and other nodes)
   </li><li>
-    Restart the config server on one of the original config server hosts and start it on the new ones,
-    one by one.
+    Restart the config server on the original config server nodes and start it on the new one,
+    one at a time (if you want to verify that new config is in use, try using
```

Comment:
As discussed, I suspect this may not work with dynamic reconfig.

Reply:
No dynamic reconfig for self-hosted anymore